build.cmd

```diff
@@ -380,6 +390,45 @@ for /f "tokens=*" %%s in ('%DotNetCli% msbuild "%OptDataProjectFilePath%" /t:Dum
     set __IbcOptDataVersion=%%s
 )
 
+REM =========================================================================================
+REM ===
+REM === Generate source files for eventing
```

Comment:
How does this affect incremental build? If I run "build.cmd" twice in a row, will it generate new files that will cause the subsequent builds to build things they shouldn't need to?

Reply:
No. I created a utility which ensures files are only written if the copy on disk is different